nbproject/project.properties

```diff
@@ -30,24 +30,23 @@ dist.jar=${dist.dir}/libai.jar
 dist.javadoc.dir=${dist.dir}/javadoc
 endorsed.classpath=
 excludes=
-file.reference.mysql-connector-5.0.5.jar=../../libs/mysql/mysql-connector-5.0.5.jar
 includes=**
 jar.compress=false
 javac.classpath=\
-    ${libs.swing-layout.classpath}:\
-    ${file.reference.mysql-connector-5.0.5.jar}
```

Comment:
Oh sorry about that, I clicked on "Resolve dependencies" and didn't realize that the connector jar was gone.

I've been looking at the `DataSet` class and it uses `javax.sql.Connection` which is driver agnostic in which case the MySQL Connector/J should be imported by whoever is using `libai` and not `libai` itself.

At any rate I will fix it now because it was an unintended change.